sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -1421,3 +1423,19 @@ def _more_tags(self):
     # to exist so that third party estimators can easily support multiple sklearn
     # versions.
     check_estimator_tags_renamed("OkayEstimator", OkayEstimator())
+
+
+# Test that set_output doesn't make the tests to fail.
+@parametrize_with_checks(
```

Comment:
We can not use `parametrize_with_checks` here because it depends on `pytest`. The simple fix is to move it to `sklearn/tests/test_common.py`.

Reply:
I kinda prefer to have `test_common.py` to have as least tests as possible, to reduce the difference between what we force on third party estimators and what we expect from our own.

So I refactored the test here.